src/app/query/common/compliance-checker/compliance-checker.query.ts

```diff
@@ -0,0 +1,75 @@
+import { type UseQueryOptions, useQueries } from '@tanstack/react-query';
+import axios from 'axios';
+
+import type { BitcoinNetworkModes } from '@shared/constants';
+import { ensureArray } from '@shared/utils';
+
+import { useAnalytics } from '@app/common/hooks/analytics/use-analytics';
+import { useCurrentNetwork } from '@app/store/networks/networks.selectors';
+
+const checkApi = 'https://api.chainalysis.com/api/risk/v2/entities';
+
+const headers = {
+  // Known public key, do not open a vulnerability report for this
+  Token: '51d3c7529eb08a8c62d41d70d006bdcd4248150fbd6826d5828ac908e7c12073',
+};
+
+async function registerEntityAddressComplianceCheck(address: string) {
+  const resp = await axios.post(checkApi, { address }, { headers });
+  return resp.data;
+}
+
+async function checkEntityAddressComplianceCheck(address: string) {
+  const resp = await axios.get(checkApi + '/' + address, { headers });
+  return resp.data;
+}
+
+interface ComplianceReport {
+  risk: 'None' | 'Low' | 'Moderate' | 'Severe';
```

Comment:
Pull this out into an enum?

Reply:
Extracted into separate string literal instead of enum